### DISCUSSION TITLE
Unable to generate 500 requests per second during load test. Receiving more than 50% of failures due to errors.EADDRINUSE

### DISCUSSION THREAD

**Discussion by rrnareshkumar** [artilleryio/artillery#2095] [Artillery]
Hi ,

I am trying to generate 500 requests per second during load test but unable to generate 500 requests per second,  Receiving more than 50% of failures due to errors.EADDRINUSE. Please refer the script(config ,scenario) and report . 

Please suggest how to avoid errors.EADDRINUSE , resolve this issue and generate 500 requests per second successfully. Please let me know if any changes are required from config or scenario to achieve this scenario.

I am using artillery version 2.0.0.35 and using windows machine.

script : 

config:
  target: https://example.com/
  phases:
    - duration: 600
      arrivalRate: 1
      rampTo: 500
      maxVusers: 500
    - duration: 1800
      arrivalRate: 500
      maxVusers: 500
scenarios:
  - flow:
      - get:
          url: /test
      - think: 1

Output /Report : 

errors.EADDRINUSE | 475319
-- | --
errors.ETIMEDOUT | 3000
http.codes.200 | 315266
http.downloaded_bytes | 4728990
http.requests | 793585
http.responses | 315266
vusers.completed | 315266
vusers.created | 793585
vusers.created_by_name.0 | 793585
vusers.failed | 478319
vusers.skipped | 256721

**Comment by hassy** [comment]
hi @rrnareshkumar 👋 Are you testing a server running on your local machine?

**Reply by rrnareshkumar** [reply]
Hi , No . Application is hosted on Cloud instance. For your information , I have tested the same API-executed few rounds of load tests ,generated 100 requests per second using artillery and I had not received any errors. Facing this issue when i try to generate 500 request/sec .
